src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price = 40 * days;
+
+  if (days >= 3) {
+    price = (40 * days) - 20;
+  }
```

Comment:
```suggestion
  if (days >= 3) {
    return price - 20;
  }
```
Don't duplicate yourself. Also, you can just immediately return the price with discount